lib/default.nix

```diff
@@ -48,15 +49,19 @@ rec {
            labelArgument =
              if (labelOption != null && label != null) then "${labelOption} '${label}'"
              else "";
+            fsCreateExtraArgsParsedToMkfs =
+              if fsCreateExtraArgs != null
+              then nixpkgs-lib.concatStringsSep " " fsCreateExtraArgs # I dunno why nixpkgs-lib.escapeShellArgs doesnt work with btrfs.
```

Comment:
Can you show your problematic `fsCreateExtraArgs` value?

Reply:
It doesn't want to see single as a parameter even though it exist. 
```
ERROR: unknown metadata profile  single
```